book.bib

```diff
@@ -317,15 +317,15 @@ @article{Gil2016
   doi = {10.1002/2015EA000136},
   journal = {Earth and Space Science},
   pages = {388-415},
-  title = {{Toward the Geoscience Paper of the Future: Best practices for documenting and sharing research from data to software to provenance}},
+  title = {{Toward the Geoscience Paper of the Future: Best Practices for Documenting and Sharing Research from Data to Software to Provenance}},
   volume = {3},
   number = {10},
   year = {2016}
 }
 
 @article{Gold1991,
   author = {David Goldberg},
-  title = {What Every Computer Scientist Should Know About Floating-Point Arithmetic},
+  title = {What Every Computer Scientist Should Know about Floating-Point Arithmetic},
```

Comment:
If we're worried about Bibtex lower-casing some words again, we could use double curly braces `{{` `}}` at the beginning and end of every title field?